### DISCUSSION TITLE
Quasar UI kit for Figma

### DISCUSSION THREAD

**Discussion by ankurgla22** [quasarframework/quasar#14351] [Show and Tell]
We are thinking to develop an app and we are considering using quasar for it.
We use Figma for design preview. 
Is there any quasar ui kit for figma ???
If not is there any future plan for its availability?

**Comment by 209** [comment]
up

**Reply by philippbroemer** [reply]
I found this thread from the old quasar forum (2019) which references using Material 2. https://forum.quasar-framework.org/topic/3197/quasar-ui-design-kit-for-sketch-and-or-adobe-xd

****Question:** Is this still accurate, does it make sense to use standard material design?**https://www.figma.com/@materialdesign (which is linked in the article).
Hope I am asking this in the right way (I am product dude, not a developer.)

**Comment by cerbenus** [comment]
Check out this one:

https://luisa.cloud/

**Comment by Zignature** [comment]
https://www.figma.com/community/file/1254101728937353537

Just a couple of components...

**Reply by adarshmadrecha** [reply]
This link is 404 now.

**Reply by Zignature** [reply]
Try this: https://www.figma.com/community/file/1379867024680261137/fiverr-ui-kit-quasar-v-2-14-1

Or this: https://www.figma.com/community/search?resource_type=mixed&sort_by=relevancy&query=quasar&editor_type=all&price=all&creators=all

**Reply by trexcodess** [reply]
These were great! But I'm looking for a quasar template or do I have to make one? If i create one then I'll upload it!

**Reply by Zignature** [reply]
Those are Quasar templates for Figma... 😕
